app/src/components/filters/Container.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <div class="row mt-3">
+        <div class="col-sm-12">
+            <h5>Filter by {{ formName }}</h5>
+            <hr />
+            <div class="accordion">
+                <div
+                    v-for="(field, fieldName, index) in fields"
+                    :key="fieldName"
+                    class="card"
+                >
+                    <div class="card-header">
+                        <h5 class="mb-0">
+                            <button
+                                :data-target="'#filter-dropdown-' + index"
+                                class="btn btn-link"
+                                type="button"
+                                data-toggle="collapse"
+                            >
+                                {{ field.field }}
+                            </button>
+                        </h5>
+                    </div>
+                    <div
+                        :id.prop="'filter-dropdown-' + index"
+                        class="collapse"
+                    >
+                        <div class="card-body">
+                            <text-filter
+                                v-if="field.fieldType == 'text'"
```

Comment:
Right now, the only filterable fields in the app are text fields. If in the future we introduce other types of filters, we can load the different types of filters using a similar `v-if="field.fieldType == '<type>'"` pattern to pull the right components.